src/docker/composebuilder/service_builder.go

```diff
@@ -148,7 +156,14 @@ func (d *DevelopmentDockerComposeBuilder) getDockerEnvVars() map[string]string {
 			result[variable] = value
 		}
 	}
-	envVars, secrets := d.ServiceConfig.GetEnvVars("development")
+	var envVars map[string]string
+	var secrets []string
+	switch d.Mode.Environment {
+	case BuildModeEnvironmentProduction:
+		envVars, secrets = d.ServiceConfig.GetEnvVars("production")
+	default:
+		envVars, secrets = d.ServiceConfig.GetEnvVars("development")
+	}
```

Comment:
Noticed we weren't building for production env vars in local production.

Reply:
I don't think we should be. Local production is only for testing the dockerfiles. @alexdavid thoughts?

Reply:
Shouldn't local production mimic as close as possible the environment in production? Currently `run_production.yml` would be technically incorrect since it compiles env vars from the dependency block of service.yml. Anyways, these changes have been extracted into a separate PR: https://github.com/Originate/exosphere/pull/793